addon-test-support/index.ts

```diff
@@ -1,5 +1,9 @@
 import { createSandbox, restoreSandbox } from './sinon-sandbox';
-
+declare global {
```

Comment:
I think we might be able to replace this with a real import, actually:

`import * as QUnit from 'qunit';`

Reply:
We would have to add `qunit` as a peerDependency of the package, like `sinon`. It probably already should have been one, to be honest.

Reply:
If this works, we'd drop the `self.` from line 11, I think.